packages/edit-post/src/components/sidebar/settings-sidebar/index.js

```diff
@@ -126,19 +126,15 @@ const SidebarContent = ( { tabName, keyboardShortcut, isEditingTemplate } ) => {
 							/>
 						}
 					/>
-					{ ! isEditingTemplate && (
-						<>
-							<PostStatus />
-							<PluginDocumentSettingPanel.Slot />
-							<PostLastRevisionPanel />
-							<PostTaxonomiesPanel />
-							<PostExcerptPanel />
-							<PostDiscussionPanel />
-							<PageAttributesPanel />
-							<PatternOverridesPanel />
-							<MetaBoxes location="side" />
-						</>
-					) }
+					{ ! isEditingTemplate && <PostStatus /> }
+					<PluginDocumentSettingPanel.Slot />
+					<PostLastRevisionPanel />
+					<PostTaxonomiesPanel />
+					<PostExcerptPanel />
+					<PostDiscussionPanel />
+					<PageAttributesPanel />
+					<PatternOverridesPanel />
```

Comment:
Do you think that all of these components being rendered for templates is fine? It probably is just want to confirm that it doesn't result in different UIs than trunk :)

Reply:
They were still rendered here: https://github.com/WordPress/gutenberg/pull/60961/files#diff-6139877d7164dad773ae6521334135a37eb7e0bb351ade4c4f2169aa69da6e34L137 in site editor, so it should be the same in post editor, since they have checks whether to render or not.

The alternative would be to separate them in site editor too(the above linked code) and just remove them from there and keep them in PagePanels.

Reply:
Additionally the revisions panel was rendered in site editor and now it does in post editor too.

Reply:
Definitely worth unifying like you did.